src/Apps/Order/Routes/Submission/index.tsx

```diff
@@ -63,6 +61,7 @@ export const SubmissionFragmentContainer = createFragmentContainer(
     fragment Submission_order on Order {
       id
       code
+      ...Summary_order
```

Comment:
@sweir27 @zephraph Interestingly enough, I found a bug in a component that was trying to [pass along data](https://github.com/artsy/reaction/blob/2805ed1baef4b2547414409ee0c5665ee86c7a4f/src/Apps/Order/Routes/Submission/index.tsx#L51) for a child component for which it hadn’t requested data in its fragment.

Reply:
Heh 😅